src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -64,25 +70,54 @@ const getTextProps = (index: number, tasksDone: number): TextProps => {
 }
 
 const textWithCopyIcon = (text: string): JSX.Element => {
+  const { onCopy, hasCopied } = useClipboard(text)
   return (
-    <Button variant="link" onClick={() => navigator.clipboard.writeText(text)}>
-      <Box display="flex" alignItems="center" maxW="16rem">
-        <Text
-          textStyle="body-2"
-          color="black"
-          whiteSpace="nowrap"
-          overflow="hidden"
-          textOverflow="ellipsis"
+    <Button variant="link" onClick={onCopy}>
+      {/* Closes after 1.5s and does not refocus on the button to avoid the outline */}
+      <Popover
+        returnFocusOnClose={false}
+        isOpen={hasCopied}
+        placement="right-start"
+      >
+        <PopoverTrigger>
+          <Box display="flex" alignItems="center" maxW="16rem">
+            <Text
+              textStyle="body-2"
+              color="black"
+              whiteSpace="nowrap"
+              overflow="hidden"
+              textOverflow="ellipsis"
```

Comment:
nit: prefer `noOfLines`

Reply:
have to check the styling - the `Text` hasn't overflowed its bounding box yet i think

Reply:
cos from the docs [here](https://chakra-ui.com/docs/components/text/usage#truncate-text), it _should_ display ellipsis (and in prior experience, it has)